### DISCUSSION TITLE
Add more output details for debug purposes

### DISCUSSION THREAD

**Discussion by ribalba** [green-coding-solutions/green-metrics-tool#339] [Ideas]
Currently we don't really have a way to add debug output to our code that is only outputted when run with --debug or sometimes I would also want to have less output. I would like the functionality to add some logging to my GMT code that is only outputted when run with --debug.

I would propose to change all print statements with an actual logger. So we can have different verbosity levels. As we don't have discussions I am using a ticket :)

**Comment by ribalba** [comment]
From @ArneTR 
I guess the sane place to add it to would be "Logs" Tab, as in example here: https://metrics.green-coding.berlin/stats.html?id=9d250a5f-1f01-42a2-926e-e3f9b216ed5a

Just the prints from the GMT should be logged separately and then just added to self.__stdout_logs

**Reply by ribalba** [reply]
I would also want different "levels" like INFO, DEBUG, ...

**Reply by djesic** [reply]
I have this logging library starred: https://github.com/Delgan/loguru
And this one is very helpful for debugging https://github.com/samuelcolvin/python-devtools
